db/migrate/20200109163351_add_invitation_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddInvitationToUser < ActiveRecord::Migration[6.0]
+  def change
+    add_column :users, :invitation, :string
+  end
+end
```

Comment:
- [ ] You are adding a column to users but apparently the first migration setting up the table has been erased